x11-misc/gmrun/gmrun-0.9.2-r2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils
```

Comment:
I do not see how `eutils` eclass serves you here :)

Reply:
removed in current version. it was from -r1 for epatch